### PR TITLE
refactor/ClearingHouseConfig-cleanup

### PR DESCRIPTION
**Pull request by tienshaoku**
… IClearingHouseConfig.sol.

### PR Reminders

Be ware of the followings

- [ ] implement deployment script for your changes in deployment repo
- [ ] add verification in hardhat simulation(000-prepare-simulation-check, 902-newMarket-check or 903-simulation-check) and system test(901-system-test) in deployment repo
- [ ] update change log
 
---

1. **Contract**: make sure the code follows our convention, ex: naming, explicit returns, etc.; if uncertain, discuss with others

2. **Test**: make sure tests can cover most normal and edge cases; if not, open follow-up tickets. Also, look out for failed tests and fix them!

3. **CHANGELOG.md**: update when external interfaces are changed

4. Workflow: 
    - Github: assign the pr to yourself; if pairing, can merge directly; else, assign someone to help review
    - Asana: assign the corresponding ticket to yourself and leave the pr link on it for easier follow-ups
    - Discord: if someone is mentioned in this pr, tag on Discord

### CODE REVIEW

File: contracts/interface/IClearingHouseConfig.sol [hunk 1]
```diff
@@ -1,7 +1,27 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 pragma solidity 0.7.6;
 
-interface IClearingHouseConfig {
+interface IClearingHouseConfigEvent {
+    event LiquidationPenaltyRatioChanged(uint24 liquidationPenaltyRatio);
+
+    event PartialCloseRatioChanged(uint24 partialCloseRatio);
+
+    event TwapIntervalChanged(uint32 twapInterval);
```
Comment (tienshaoku) [inline contracts/interface/IClearingHouseConfig.sol]: One thing worth noticing is that the param used to be `uint256 twapInterval` but it should be `uint32` so I changed it here.
Not 100% sure but I think this won't break anything and would love to be corrected if this isn't true!

Comment (tailingchen) [reply contracts/interface/IClearingHouseConfig.sol]: Change type of param will derive different topic of event.
Need to check if it's used in subgraph.

**Review by tailingchen — APPROVED** [review]
